development/utils/git-commit-hash.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+module.exports = () =>
+  require('child_process')
+    .execSync('git rev-parse --short HEAD', {
```

Comment:
```suggestion
module.exports = (short) =>
  require('child_process')
    .execSync(`git rev-parse ${short? : '--short' : ''} HEAD`, {
```
May be redundant, but I think that using the short version apart from when "human reading" is potential for conflict down the line...